docroot/profiles/custom/cgov_site/modules/custom/cgov_image/cgov_image.module

```diff
@@ -78,15 +78,18 @@ function cgov_image_media_view(array &$build, EntityInterface $entity, EntityVie
  *   The module may add elements to $build prior to rendering.
  *   The structure of $build is a renderable array as expected by
  *   drupal_render().
- * @param Drupal\Core\Entity\EntityInterface $entity
+ * @param \Drupal\Core\Entity\EntityInterface $entity
  *   The entity object.
- * @param Drupal\Core\Entity\Display\EntityViewDisplayInterface $display
+ * @param \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display
  *   The entity view display holding the display options configured for
  *   the entity components.
  * @param string $view_mode
  *   The view mode the entity is rendered in.
  */
 function _cgov_image_add_enlarge_link(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
+  if(!($entity instanceof MediaInterface)){
```

Comment:
You should change the signature of the function to be a MediaInterface; then in the calling function do the check. This is because this is not a hook that is forced to use the most generic of interfaces.